beta/package.json

```diff
@@ -100,6 +100,6 @@
   },
   "lint-staged": {
     "*.{js,ts,jsx,tsx,css}": "yarn prettier",
-    "*.md": "yarn fix-headings"
+    "!(*README|CONTRIBUTING).md": "yarn fix-headings"
```

Comment:
Why * before README?

Reply:
Can we just scope it down to src/**/*.md?

Reply:
We can do that :)